src/sitecustomize.py

```diff
@@ -1,9 +0,0 @@
-"""
-Pyodide-specific startup code that's always run at Python startup.
-"""
-
-# We weren't invoked at the commandline, but set some commandline arguments as
-# if we were anyway. Helps many of the CPython tests pass.
-import sys
-
-sys.argv = ["pyodide"]
```

Comment:
Yes, we can remove these two lines, however I wonder it it would make sense to keep the file. It's a standard mechanism to execute install specific code at startup, which might be better than doing it at import. For instance, if we want to monkeypatch some stdlib modules to have different behavior this would be the place for it IMO.